### ISSUE TITLE
[Bug] Preview, if cursor not hover to image

### ISSUE THREAD

**Issue by Kristinita** [alvesjtiago/hover-preview#8]
### 1. Summary

I can see preview, if I don't hover my cursor to image.

### 2. Expected behavior

Image preview only if my cursor hover to image.

### 3. Actual behavior

![Preview without hover](http://i.imgur.com/qx9P2gs.png)

### 4. Settings

I have file `test.md`. Content of my file:

```markdown

Sasha One in A Million









![Sasha Image](http://i.imgur.com/Dt0149I.png)
```

### 5. Steps to reproduce

The problem is reproduced for me in a version of Sublime Text without plugins and user settings.

In my file `test.md` I hover my cursor to `Sasha` word → I see preview of my image the link below. My carriage also don't situated in image URL.

### 6. Environment

**Operating system and version:**
Windows 10 Enterprise LTSB 64-bit EN
**Sublime Text:**
Build 3126

Thanks.

**Comment by alvesjtiago** [comment]
@Kristinita just released version 0.0.7 that fixes this issue. Let me know if it works. Thank you

**Comment by Kristinita** [comment]
@alvesjtiago , thanks, now I can get no this bug! We wait, when we can preview images in a larger size.
